dash_core_components/package.json

```diff
@@ -8,64 +8,64 @@
   },
   "main": "src/index.js",
   "scripts": {
-    "build-dev": "builder run clean-lib && builder run extract-metadata && builder run generate-python-classes && webpack -w --config=./config/webpack/webpack.config.dev.js",
-    "build-dist": "builder run clean-lib && builder run extract-metadata && builder run generate-python-classes && cross-env NODE_ENV=production webpack --config=./config/webpack/webpack.config.dist.js",
-    "copy-lib": "cp -f lib/* dash_core_components",
     "generate-python-classes": "python -c \"import dash; dash.development.component_loader.generate_classes('dash_core_components', 'dash_core_components/metadata.json');\"",
-    "install-local": "npm run copy-lib && python setup.py install",
-    "prepublish": "npm test && builder run build-dist && npm run copy-lib",
-    "publish-all": "npm publish && python setup.py sdist upload",
-    "publish-pypi": "npm run prepublish && python setup.py sdist && twine upload --sign --skip-existing",
-    "start": "./node_modules/.bin/builder run build-dev",
-    "test": "eslint src",
-    "test-watch": "./node_modules/.bin/builder run test-frontend-watch",
-    "test-debug": "./node_modules/.bin/builder run test-frontend-debug",
+    "prepublish": "npm run build:js && npm run build:py",
+    "publish-all": "node scripts/publish.js",
+    "start": "webpack-serve ./webpack.serve.config.js --open",
+    "lint": "eslint src",
+    "test": "python -m unittest test.test_integration",
+    "format": "prettier --config .prettierrc --write src/**/*.js",
     "uninstall-local": "pip uninstall dash-core-components -y",
     "build:js": "webpack --mode production",
-    "build:py": "node ./extract-meta src/components > dash_core_components/metadata.json && cp package.json dash_core_components && npm run generate-python-classes"
+    "build:js-dev": "webpack --mode development",
+    "build:py": "node ./extract-meta src/components > dash_core_components/metadata.json && cp package.json dash_core_components && npm run generate-python-classes",
+    "build:all": "npm run build:js && npm run build:py",
+    "build:all-dev": "npm run build:js-dev && npm run build:py",
+    "build:watch": "watch 'npm run build:all' src"
```

Comment:
should this be `build:all-dev` since we'd most likely use the `watch` while we're in development?

Reply:
Yeah good point, I wasn't sure about that. Having `build:js-dev` has some value if you just want to run the JS side (although, what does the `development` flag actually do for us here? It's not watching anything or rebuilding or hot-reloading etc) but I guess `build:all-dev` doesn't really have that much value. I wondered however, if `build:all-dev` makes it clear enough that you're running a watcher, as opposed to `build:watch` makes it pretty clear that you are. My suggestion would be to drop the `build:all-dev` altogether, and keep `build:watch`.

Reply:
I use `build:js-dev` in #293, also added to the prepublish command because we need the dev bundle in the package.